.travis.yml

```diff
@@ -16,4 +16,6 @@ install:
 - bundle install
 
 script:
-- bundle exec jekyll build
+- bundle exec jekyll build 2> error.log
+- cat >&2 error.log
+- grep -qie Error error.log && false || true
```

Comment:
You're right. (http://stackoverflow.com/questions/17203122/bash-if-else-statement-in-one-line#comment33428887_17203203)

I recommend instead using:

```bash
[ "$(grep -cie Error error.log)" -eq 0 ]
```